src/gui/editorwidgets/qgsvaluerelationwidgetwrapper.cpp

```diff
@@ -131,7 +131,11 @@ void QgsValueRelationWidgetWrapper::initWidget( QWidget *editor )
     mTableWidget->setEditTriggers( QAbstractItemView::NoEditTriggers );
     mTableWidget->setSelectionMode( QAbstractItemView::NoSelection );
     if ( mCache.size() > 0 )
-      mTableWidget->setRowCount( ( mCache.size() + config( QStringLiteral( "NofColumns" ) ).toInt() - 1 ) / config( QStringLiteral( "NofColumns" ) ).toInt() );
+    {
+      const int nofCols = config( QStringLiteral( "NofColumns" ) ).toInt();
+      const int denom = nofCols != 0 ? nofCols : 1;
+      mTableWidget->setRowCount( ( mCache.size() + nofCols - 1 ) / denom );
```

Comment:
Is this related?

Reply:
No it's not related but I came across this issue with my testing project. If you prefer, I can create a dedicated PR?

Reply:
Ah no, that looks ok.